src/core/validator.ts

```diff
@@ -1,83 +1,82 @@
 import chalk from 'chalk';
-import FieldLoader from '../util/fieldLoader';
-import Differ from '../util/differ';
-import GithubFetcher from '../util/githubFetcher';
-import Report from '../util/report';
+import fieldLoader from '../util/fieldLoader';
```

Comment:
util folder seems to be abused here to have business logic. 
This seems like business logic rather than util classes

Reply:
We use the `fieldLoader` util just to load JSON fields from disk or URL. If you think it's business logic cause we're fetching things from GitHub then I guess you're right but we treat it as a utility thing really 🤷‍♂️.

Reply:
I think in that specific case of `fieldLoader` was part of the growth plan of oratrix. Imagine others features also fetching the fields from disk and URL. To avoid duplicate the code, it lives in a separated file inside `util` dir. But due the changes, maybe it is not required anymore. So we can be resilient and flexible to move the parts with no problem. Also the `report` is the same situation because the `licenses` feature was planned to use cli-table3.